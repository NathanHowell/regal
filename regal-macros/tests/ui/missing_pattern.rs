use regal_macros::RegalLexer;

#[derive(RegalLexer)]
enum Bad {
    A,
    #[token("a")]
    B,
}

fn main() {}
