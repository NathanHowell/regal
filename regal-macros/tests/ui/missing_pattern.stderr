error: Each variant must have a #[token(..)] or #[regex(..)] attribute
 --> tests/ui/missing_pattern.rs:5:5
  |
5 |     A,
  |     ^
